api/db/changelog/db.changelog-xxx-remove-old-workspace-columns.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="thibault" id="[updateme]">
+    <dropColumn tableName="workspace" columnName="billing_account_type"/>
+<!-- wait for Eric
+    <dropColumn tableName="workspace" columnName="billing_status"/>
+-->
+    <dropColumn tableName="workspace" columnName="data_access_level"/>
+    <dropColumn tableName="workspace" columnName="needs_rp_review_prompt"/>
+<!-- wait for 8752
+    <dropColumn tableName="workspace" columnName="published"/>
+-->
+      </changeSet>
+    </databaseChangeLog>
```

Comment:
formatting of this file seems a bit off...

is there another file needed for liquibase migration? for my own education, does AoU DB migration happens as part of deployment?

Reply:
this isn't ready yet :)

Yes, deployment and `dev-up` both run a liquibase step which runs files like this